### PR TITLE
Support for blacklight 5.10

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by cam156** [comment]
@jcoyne Is there a reason these two commits are submitted together?

File: Gemfile [hunk 1]
```diff
@@ -13,6 +13,7 @@ group :development, :test do
   gem 'coveralls', require: false
   gem 'rspec-activemodel-mocks'
 end
+gem "blacklight", github: 'projectblacklight/blacklight'
```
Comment (cam156) [inline Gemfile]: Is this line needed?  I would have thought the gemspec would be enough

**Comment by jcoyne** [comment]
@cam156 they are submitted together because the second one depends on the first on. I could rebase them, but I've already got the first commit in production and if I do that it causes problems when you  `bundle install`

File: Gemfile (hunk 1 above)
Comment (jcoyne) [inline Gemfile]: Yes, because blacklight 5.10 hasn't been released yet.

**Comment by cam156** [comment]
@jcoyne Ok.  :+1: Two commits!

**Comment by jcoyne** [comment]
@cam156 I updated it to `some_rows` because your suggestion `100_rows` is not a valid method name.

**Pull request merged**